Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM zonemaster/engine:local as build
+
+ARG version
+
+RUN apk add --no-cache \
+    # Only needed for Net::Interface
+    build-base \
+    perl-dev \
+    perl-doc \
+    make \
+    # Compile-time dependencies
+    perl-app-cpanminus \
+    perl-module-install \
+ && cpanm --no-wget \
+    MooseX::Getopt \
+    Text::Reflow
```

Comment:
Calling _cpanm_ like that, won't that download the packages using **http://** rather than the presumably preferred **https://**?

Reply:
You're absolutely right! I wasn't aware of this. Thanks for pointing it out!